docassemble/AssemblyLine/data/questions/al_package_unstyled.yml

```diff
@@ -1,20 +1,4 @@
-comment: |
-  The purpose of this YAML file is to allow you to make use of all of the Assembly
-  Line standard questions, etc. with one include, but eliminate all code that 
-  changes the visual appearance of your interview.
+# DEPRECATED - use assembly_line_unstyled instead
```

Comment:
Don't think you included this file in the branch yet

Reply:
Everytime, you fix it the moment I comment. :laughing:

Reply:
Caught it at the same moment again :)

Reply:
I think I have a bad habit of opening the PR to double-check what the branch contains. I should just use compare without clicking the "open PR" button.